frontend/src/Containers/Home/homeSaga.js

```diff
@@ -90,6 +100,10 @@ export function* fetchAllUsaStateOptions() {
   }
 }
 
+export function* watchFetchUsername() {
+  yield takeLatest(`home/getUsername`, fetchUsername);
```

Comment:
no need to have tilda rather lets use regular `''` since there is no variables here.

Reply:
it is like this all around. lets keep it like such.